handshake_manager.go

```diff
@@ -145,6 +145,8 @@ func (c *HandshakeManager) handleOutbound(vpnIp iputil.VpnIp, f udp.EncWriter, l
 
 	// Get a remotes object if we don't already have one.
 	// This is mainly to protect us as this should never be the case
+	// NB ^ This comment doesn't jive. It's how the thing gets intiailized.
```

Comment:
NB = Nota Bene, a note I left for myself when reading the code.
NB != Nate Brown, that's just a fun coincidence.